docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -72,12 +72,12 @@ space. While the exact amount resource usage of each additional field depends
 on the type of field, an additional 1024B in memory use  for each additional
 mapped field on data nodes is a good approximation in most cases.
 As a result, the more mapped fields you have per index, the fewer
-indices and shards will each individual data node be able to hold,
+indices and shards each individual data node will be able to hold,
 all other things being equal. In addition to the per field overhead you should
 at least have an additional 512MB of heap available per data node.
 For example, holding 1000 indices that each contain 4000 fields will require
-1000 x 4000 x 1024b = 4Gb for the fields and another 512Mb for a total of
-at least 4.5Gb on the data node.
+1000 × 4000 × 1024B = 4GB for the fields and another 512MB for a total of
+at least 4.5GB on the data node.
```

Comment:
```suggestion
at least 4.5GB heap or 9GB RAM on the data node.
```

I would prefer we call out the heap vs RAM carefully here.

Reply:
+1 to mentioning heap, but I would rather not imply that RAM is always twice the heap. It's the heap size that matters here I think.